packages/financial-templates-lib/src/clients/ExpiringMultiPartyClient.js

```diff
@@ -1,19 +1,32 @@
 // A thick client for getting information about an ExpiringMultiParty. Used to get sponsor information, outstanding
 // positions, undisputed Liquidations, expired liquidations, disputed liquidations.
 
-const { ConvertDecimals, parseFixed, LiquidationStatesEnum } = require("@uma/common");
+const { ConvertDecimals, LiquidationStatesEnum } = require("@uma/common");
 const Promise = require("bluebird");
-
 class ExpiringMultiPartyClient {
```

Comment:
The core logic for dealing with decimals now takes place within this `ExpiringMultiPartyClient`. Previously, the price feed was scaled according to decimals of the collateral type to accommodate BTC collateral currencies. After this PR the Client takes in all three decimals and uses them within the computation to calculate position collateralization.